### DISCUSSION TITLE
使用油猴脚本【字体渲染】优化简悦的阅读模式字体

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#4465] [使用技巧]
> [字体渲染](https://greasyfork.org/zh-CN/scripts/416688-%E5%AD%97%E4%BD%93%E6%B8%B2%E6%9F%93-%E8%87%AA%E7%94%A8%E8%84%9A%E6%9C%AC) 是一款基于 Tampermonkey 油猴脚本，可以优化浏览器阅读界面的字体和相关设置。

# 受众

喜欢折腾字体的用户，尤其是 Windows 用户（可以以肉眼可见的效果改善 Windows 浏览器效果使用效果不佳的原因。

# 截图

> 下面的截图使用了 **更纱黑体**

![image](https://user-images.githubusercontent.com/81074/188547468-b8277185-b0ae-49b3-8316-3cc667fd1637.png)

# 安装

需要先安装 [Tampermonkey](https://chrome.google.com/webstore/detail/tampermonkey/dhdgffkkebhmkfjojejmpbldmpobfkfo)， 然后再安装 [字体渲染](https://greasyfork.org/zh-CN/scripts/416688-%E5%AD%97%E4%BD%93%E6%B8%B2%E6%9F%93-%E8%87%AA%E7%94%A8%E8%84%9A%E6%9C%AC)  即可。

# 使用

安装后随便进入一个页面 e.g https://sspai.com/post/69972 不要进入简悦的阅读模式，打开

![image](https://user-images.githubusercontent.com/81074/188547876-f91ae5f0-be4b-4b62-bb08-cece0bed7ec9.png)

然后选择一个本机存在的字体

![image](https://user-images.githubusercontent.com/81074/188548033-e9ed77e0-5452-40a3-800b-d648f471a18d.png)

保存后会提示刷新，自动刷新页面后，再进入简悦的阅读模式，会发现已经可以生效了。

同时支持安装其他内置字体或自定以字体，因为这部分可去 https://github.com/F9y4ng/GreasyFork-Scripts/discussions/46 查看详细操作。

# 附录

## 字体渲染的官网

https://github.com/F9y4ng/GreasyFork-Scripts

## 内置字体（需下载）

> 来自 https://github.com/F9y4ng/GreasyFork-Scripts/discussions/46#discussion-3740931 ，请以此链接为准。

- [更纱黑体 SC-Sarasa Gothic SC.[YES].ttf.zip](https://github.com/F9y4ng/GreasyFork-Scripts/files/7710547/SC-Sarasa.Gothic.SC.YES.ttf.zip)

- [OPPOSans-OPPOSans.[YES].ttf.zip](https://github.com/F9y4ng/GreasyFork-Scripts/files/7710535/OPPOSans-OPPOSans.YES.ttf.zip)

- [霞鹜文楷-LXGW WenKai.[YES].ttf.zip](https://github.com/F9y4ng/GreasyFork-Scripts/files/9040037/-LXGW.WenKai.YES.ttf.zip)

- [浪漫雅圆-LMYY.[YES].ttf.zip](https://github.com/F9y4ng/GreasyFork-Scripts/files/7710584/-LMYY.YES.ttf.zip)

- [苹方-简-PingFang SC.[YES].ttf.zip](https://github.com/F9y4ng/GreasyFork-Scripts/files/7710587/-.-PingFang.SC.YES.ttf.zip)

- [鸿蒙黑体-HarmonyOS Sans SC.[YES].ttf.zip](https://github.com/F9y4ng/GreasyFork-Scripts/files/7710582/-HarmonyOS.Sans.SC.YES.ttf.zip)

- [思源黑体-Source Han Sans SC.[YES].otf.zip](https://github.com/F9y4ng/GreasyFork-Scripts/files/7710599/-Source.Han.Sans.SC.YES.otf.zip)

- [思源宋体-Source Han Serif SC.[YES].otf.zip](https://github.com/F9y4ng/GreasyFork-Scripts/files/8352162/-Source.Han.Serif.SC.YES.zip)

- [汉仪旗黑-HYQiHei.[YES].ttf.zip](https://github.com/F9y4ng/GreasyFork-Scripts/files/8129924/-HYQiHei.YES.ttf.zip)

- [兰亭黑-简-Lantinghei SC.[YES].ttc.zip](https://github.com/F9y4ng/GreasyFork-Scripts/files/8196154/-.-Lantinghei.SC.YES.ttc.zip)

- [冬青黑体简-Hiragino Sans GB.[YES].otf.zip](https://github.com/F9y4ng/GreasyFork-Scripts/files/8196214/-Hiragino.Sans.GB.YES.otf.zip)

- [文泉驿微米黑-WenQuanYi Micro Hei.[YES].ttc.zip](https://github.com/F9y4ng/GreasyFork-Scripts/files/8196256/-WenQuanYi.Micro.Hei.YES.ttc.zip)

- [文泉驿正黑-WenQuanYi Zen Hei.[YES].ttc.zip](https://github.com/F9y4ng/GreasyFork-Scripts/files/8204504/-WenQuanYi.Zen.Hei.YES.ttc.zip)

- [手书体-ShouShuti.[YES].ttf.zip](https://github.com/F9y4ng/GreasyFork-Scripts/files/8352239/-ShouShuti.YES.ttf.zip)

## 更多使用帮助

https://github.com/F9y4ng/GreasyFork-Scripts/discussions

**Comment by LordHumphrey** [comment]
这个脚本和简悦的字体冲突吗？如何处理二者的关系，我这里似乎和简悦冲突啊

**Reply by Kenshin** [reply]
字体渲染的优先级好，会让简悦的字体设置无效。

注意：这不是 Bug 本该如此，否则字体渲染如何生效？
